types/react-navigation/react-navigation-tests.tsx

```diff
@@ -45,12 +45,11 @@ interface StartScreenNavigationParams {
     s: string;
 }
 
-interface StartScreenProps extends NavigationScreenProps<StartScreenNavigationParams> { }
```

Comment:
Removing an interface might break already live systems. I suggest to use type instead.
```ts
type StartScreenProps = NavigationScreenProps<StartScreenNavigationParams>;
```

Reply:
Fortunately this is a test file, not published.

Reply:
Haven't noticed it 😅.